src/scss/_variables.scss

```diff
@@ -0,0 +1 @@
+$body-font-size-base: 14px;
```

Comment:
This will be a global css variables files we have to include into all the scss files we work on or create.